### PR TITLE
changelog: add entries for 1.10.9, 1.11.6, 1.12.2

### PR DESCRIPTION
**Pull request by calvn**
This PR adds changelog entries for the Vault 1.10.9, 1.11.6, and 1.12.2 releases.

TODO:
- [x] Change "Unreleased" to actual release dates.

### CODE REVIEW

**Comment by mladlow** [comment]
We should update the 1.13 preview too - you can do this on the `main` branch using the SHA for 1.13 in the table on the Changelog wiki page.

**Comment by calvn** [comment]
Updates for 1.13.0 done in [6f0a450](https://github.com/hashicorp/vault/pull/18144/commits/6f0a450a3d2318cea8960b086355e8c05aa153ab).

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,66 +5,110 @@ CHANGES:
...
 * plugins: `GET /sys/auth/:path/tune` and `GET /sys/mounts/:path/tune` endpoints may now return an additional `plugin_version` field in the response data if set. [[GH-17167](https://github.com/hashicorp/vault/pull/17167)]
 * plugins: `GET` for `/sys/auth`, `/sys/auth/:path`, `/sys/mounts`, and `/sys/mounts/:path` paths now return additional `plugin_version`, `running_plugin_version` and `running_sha256` fields in the response data for each mount. [[GH-17167](https://github.com/hashicorp/vault/pull/17167)]
 * secrets/aws: do not create leases for non-renewable/non-revocable STS credentials to reduce storage calls [[GH-15869](https://github.com/hashicorp/vault/pull/15869)]
 * ui: Upgrade Ember to version 4.4.0 [[GH-17086](https://github.com/hashicorp/vault/pull/17086)]
-
 FEATURES:
 
-* core: Add user lockout field to config and configuring this for auth mount using auth tune to prevent brute forcing in auth methods [[GH-17338](https://github.com/hashicorp/vault/pull/17338)]
+* logging: Vault Agent supports logging to a specified file path via environment variable, CLI or config [[GH-17841](https://github.com/hashicorp/vault/pull/17841)]
```
Comment (mladlow) [inline CHANGELOG.md:19]: @calvn could you reach out to all the people on these linked PRs (I think it's mostly my team) and ask them to either move these out of the "Features" section or use the "Features" formatting? It looks like at least the logging ones might be more improvements. This isn't in the checklist but I try to keep on top of the reminders.

If people change the changelog files, this section will be automatically repaired when we re-gen the changelog, so for this PR it's not a problem to have them formatted like this.

**Review by mladlow — APPROVED** [review]
Approved though going to throw in a reminder to update the release date (please look at the last round for the format)

File: CHANGELOG.md (hunk 1 above)
Comment (calvn) [reply CHANGELOG.md:19]: cc: @ncabatoff @peteski22 @hashishaw @hellobontempo on updating the format for these entries on the features section of the CL.

Comment (peteski22) [reply CHANGELOG.md:19]: I've raised a PR for this to resolve the log related PRs. It's set to auto merge when the build completes successfully. Thanks for pointing this out, as I learned something new today. :)

Comment (hellobontempo) [reply CHANGELOG.md:19]: This thread was lost in my email! Apologies - Opened PR #18182 to address

Comment (mladlow) [reply CHANGELOG.md:19]: @hellobontempo this change is part of the 1.13 preview and isn't urgent to address. As long as they're cleaned up by the next release we'll regenerate this part of the changelog (just the 1.13 preview portion), and they'll show up in the correct section then.

Comment (hellobontempo) [reply CHANGELOG.md:19]: ah thanks for clarifying! 😄

**Pull request merged**